src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -221,8 +223,37 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
                     TEJobSet jobSet = optSet.get();
                     Currency defaultCurrency = totalEconomy.getDefaultCurrency();
 
-                    for (TEActionReward actionReward : jobSet.getActionRewards()) {
-                        lines.add(Text.of(TextColors.GOLD, "[", jobManager.titleize(actionReward.getAction()), "] ", TextColors.GRAY, actionReward.getTargetID(), TextColors.GOLD, " (", actionReward.getExpReward(), " EXP) (", defaultCurrency.format(actionReward.getMoneyReward()), ")"));
+                    for (TEAction action : jobSet.getActions()) {
+                        EconomyService service = Sponge.getServiceManager().provideUnchecked(EconomyService.class);
+                        Optional<Currency> c = Optional.empty();
+                        Text listText;
+
+                        if (action.isIDTraited()) {
```

Comment:
I can see your point but I don't quiet agree with you as mods sometimes use similar or same names or confusing ids (for both ids and traits). In this case I'd rather give too specific information rather than confusing.